node/src/chain_spec/turing.rs

```diff
@@ -0,0 +1,188 @@
+use hex_literal::hex;
```

Comment:
just guessing at this, but prob good to outline the differences to note between this and neumann. lemme know what i missed:
1. endowed accounts
2. genesis amounts for each account
3. parachain ID
4. SS58 format
5. token symbol
6. point to kusama rather than rococo

Reply:
but guessing functionality is the same?

Reply:
everything but 4 is different, i wrote this down on my checlist